src/kx122/kx122.c

```diff
@@ -223,6 +236,13 @@ kx122_context kx122_init(int bus, int addr, int chip_select_pin)
       kx122_close(dev);
       return NULL;
     }
+
+    if (mraa_i2c_frequency(dev->i2c, bus_frequency)){
```

Comment:
mraa_i2c_frequency takes a mraa_i2c_mode_t enum which may cause some confusion since the bus_frequency parameter is overloaded for spi/i2c.

Reply:
Sorry, my mistake.
I will change bus_frequency to spi_bus_frequency and have it default to -1, which will use the previous value of 10000Hz.
For I2C, I will not try to set the frequency.